lxd/storage/backend_lxd.go

```diff
@@ -3344,6 +3347,9 @@ func (b *lxdBackend) CreateCustomVolumeFromCopy(projectName string, srcProjectNa
 
 			return nil
 		}, nil)
+		if err != nil {
```

Comment:
Good catch, I'll also add to my https://github.com/lxc/lxd/pull/10423 as more appropriate there.